grakn-core/src/main/java/ai/grakn/graql/admin/ReasonerQuery.java

```diff
@@ -72,7 +72,7 @@
     /**
      * @param type the class of {@link Atomic} to return
      * @param <T> the type of {@link Atomic} to return
-     * @return stream of atoms of specified type defined in this query
```

Comment:
why are we changing `stream` to `output` in comments? did search-replace went too far?

Reply:
When using intellij's refactoring tool (not just changing the code by "search and replace"), it highlights to you the blocks of code that will automatically be refactored when executed. I missed this block that I should have removed. I will patch this.